app/assets/stylesheets/crayons.scss

```diff
@@ -32,5 +32,6 @@
 @import 'components/tabs';
 @import 'components/tags';
 @import 'components/tooltips';
+@import 'components/autocomplete.scss';
```

Comment:
small nitpick: I think these are sorted by name (we can also remove the extension)